documentation/help.md

```diff
@@ -658,52 +658,123 @@ features/commands of the Editor. To display the hotkeys just place the
-| `Alt+h`        | Flip horizontally                                                                                          |
-| `Ctrl+g`       | S-Group tool / Data S-Group tool                                                                           |
-| `Ctrl+d`       | Align and select all S-Group data                                                                          |
-| `Ctrl+r`       | Switching between the R-Group Label Tool/R-Group Fragment Tool/Attachment Point Tool                       |
-| `Ctrl+Shift+r` | R-Group Fragment Tool                                                                                      |
-| `Ctrl+Del`     | Clear canvas                                                                                               |
-| `Ctrl+o`       | Open                                                                                                       |
-| `Ctrl+s`       | Save As                                                                                                    |
-| `Ctrl+z`       | Undo                                                                                                       |
-| `Ctrl+Shift+z` | Redo                                                                                                       |
-| `Ctrl+x`       | Cut selected objects                                                                                       |
-| `Ctrl+c`       | Copy selected objects (not available for Simple objects and Text objects in IE)                            |
-| `Ctrl+m`       | Copy selected objects as Mol (not available for Simple objects and Text objects)                           |
-| `Ctrl+Shift+k` | Copy selected objects as Ket                                                                               |
-| `Ctrl+Shift+f` | Copy selected objects as Image                                                                             |
-| `Ctrl+v`       | Paste selected objects                                                                                     |
-| `+`            | Zoom In                                                                                                    |
-| `-`            | Zoom Out                                                                                                   |
-| `Ctrl+l`       | Layout                                                                                                     |
-| `Ctrl+Shift+l` | Clean Up                                                                                                   |
-| `Ctrl+p`       | Calculate CIP                                                                                              |
-| `?`            | Help                                                                                                       |
+**General**
+| Shortcut | Action | Description |
+| --- | --- | --- |
+| Mod+Delete | Clear Canvas | - |
```

Comment:
Does it make sense to explain what 'Mod' means?
Is it Ctrl/Cmd depending on OS?

Reply:
Yes, good point! It is Cmd/Ctrl depending on OS, will add to description at the top